### PR TITLE
Issue #12267: Resolve Pitest suppression for SeperatorWrapCheck

### PR DESCRIPTION
**Pull request by Kevin222004**
Issue #12267: Resolve Pitest suppression for SeperatorWrapCheck

### CODE REVIEW

**Comment by Kevin222004** [comment]
Killed Mutation report
https://kevin222004.github.io/SeperatorWrap/com.puppycrawl.tools.checkstyle.checks.whitespace/SeparatorWrapCheck.java.html

**Review by rnveach — APPROVED** [review]
Ok to merge when CI passes and no conflicts.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapSetOptionTrim.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+package com.puppycrawl.tools.checkstyle.checks.whitespace.separatorwrap;
+
+import java.util.Arrays;
+
+public class InputSeparatorWrapSetOptionTrim {
+
+    void Method() {
+        String[] stringArray = { "Barbara", "James", "Mary", "John",
+            "Patricia", "Linda" }; // violation below ''::' should be on a new line'
+        /* violation */ Arrays.sort(stringArray, String::
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapSetOptionTrim.java:18]: As you create new file.
Please make it with indentation of +2, not a +4.
Remove this leading comment.
Move trailing comment to same line where violation

**Review by romani — CHANGES_REQUESTED** [review]
Items

**Review by rnveach — APPROVED** [review]

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**